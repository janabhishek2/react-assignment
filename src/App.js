import React from "react";
import WeatherApp from "./components/WeatherApp";
import { CityContextProvider } from "./context/cityContext";
import 'public/assets/css/app.css';

function App() {
    return (
        <CityContextProvider>
            <WeatherApp />
        </CityContextProvider>
    );
}

export default App;
