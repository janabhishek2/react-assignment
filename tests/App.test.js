import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import App from "app/App.js";
import toJson from "enzyme-to-json";

configure({ adapter: new Adapter() });

describe("App render", () => {
  test("shallow render App componentn", () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
