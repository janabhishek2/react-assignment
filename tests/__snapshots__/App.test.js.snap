// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render shallow render App componentn 1`] = `
<CityContextProvider>
  <WeatherApp />
</CityContextProvider>
`;
